_templates/api-story-card/with-prompt/src/components/index.ejs.t

```diff
@@ -14,6 +13,7 @@ import api from "../../state/<%=slug%>/api";
 const <%=StoryCardName%> = ({ init, data, Layout }) => {
   useEffect(() => {
     init();
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
>> https://github.com/facebook/create-react-app/issues/6880

Reply:
I think it would take me awhile to figure out what to do in this case and isn't the best use of my time currently. But there is the chance that I don't understand the danger well enough, so if you think that's the case let me know and I will research.

Desired effect: `init()` is run when the component mounts (like `componentDidMount` in older react). The `[]` second argument is used to prevent the function from being called more than once. Once the data has been fetched, it never needs to be fetched again.